submissions/keiran-vien/scripts/article.js

```diff
@@ -23,6 +23,14 @@
   // TODO:DONE Set up a DB table for articles.
   Article.createTable = function(callback) {
     webDB.execute(
+      'DROP TABLE articleData;', // what SQL command do we run here inside these quotes?
+      function(result) {
+        console.log('fuck the table', result);
```

Comment:
Not super professional. Fine for debug, but take it out before you commit and push.